userspace/libscap/engine/savefile/converter/converter.cpp

```diff
@@ -366,6 +366,8 @@ static conversion_result convert_event(std::unordered_map<uint64_t, safe_scap_ev
 	PRINT_EVENT(new_evt, PRINT_HEADER);
 
 	scap_evt *tmp_evt = NULL;
+	// If this is true at the end of the for loop we will free its memory.
```

Comment:
Yep it makes sense, the PR description explains well why we need it

Reply:
Thank you for your precious help!